addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -185,19 +185,18 @@ class WebsiteCrmPartnerAssign(WebsitePartnerPage):
     def sitemap_partners(env, rule, qs):
         if not qs or qs.lower() in '/partners':
```

Comment:
Just a question to be sure I am testing correctly: we agree that to see the updated sitemap, we have to delete the previous one (in Settings (debug mode) > Technical > Attachments) and then refresh before going back to `/sitemap.xml` ? :slightly_smiling_face:

Reply:
@sobo-odoo Yes, or wait 12 hours :joy: (see `SITEMAP_CACHE_TIME`)